css/files.css

```
@@ -1,4 +1,10 @@
 #body-user {
+	.files-list__filters {
+		display: none;
+	}
+	.files-list__thead{
+		top: 0;
+	}
```

```suggestion
	.files-list__thead, .files-list__thead-overlay {
		top: 0;
	}
```

This should fix the wrongly positioned table header when a file or folder is selected.

---

Thank you finding! Proposal adopted.